react-native-libraries.json

```diff
@@ -10505,5 +10505,14 @@
     "images": ["https://github.com/AnkiPro/react-native-window-resize/raw/main/assets/example.gif"],
     "ios": true,
     "dev": true
+  },
+  {
+    "githubUrl": "https://github.com/iChengbo/react-native-imagemin-asset-plugin",
+    "npmPkg": "react-native-imagemin-asset-plugin",
```

Comment:
```suggestion
```

We can also skip npm package name since it matches repository name. The filed is mostly tailored for use with scoped packages.

Reply:
Hi @Simek , thank you for your suggestion and I have made changes.